File: FrontEnd (C#)/SoftProgPersistance/GestMaterial/DAO/BibliotecaDAO.cs
using SoftProgModel.GestMaterial;
using SoftProgPersistance.DAO;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftProgPersistance.GestMaterial.DAO
{
    public interface BibliotecaDAO : IDAO<Biblioteca>
    {
        BindingList<Biblioteca> listar_bibliotecas_por_material(int _id_material);
    }
}

File: BackEnd (Java)/BibliotecaDA/src/main/java/biblioteca/gestionMaterial/dao/BibliotecaDAO.java
package biblioteca.gestionMaterial.dao;

import biblioteca.dao.IDAO;
import java.util.ArrayList;
import pe.edu.pucp.utilsarmy.gestion_de_material.model.Biblioteca;

public interface BibliotecaDAO extends IDAO<Biblioteca>{
    ArrayList<Biblioteca> listar_bibliotecas_por_material(int _id_material);
}

